app/admin/order.rb

```diff
@@ -0,0 +1,12 @@
+ActiveAdmin.register Order do
+  index do
+    column :user
+    column :total
+    column "Line Items" do |order|
+      order.line_items.each do |line_item|
```

Comment:
@Carrigan thanks for the review!!

Attached two screenshots - without the each loop, I get one object, whereas with the loop I get the data of each line item. 

However, what I really want is to print out the subcolumns, like line_item.quantity. Any tips?
<img width="743" alt="screen shot 2016-02-04 at 8 43 42 am" src="https://cloud.githubusercontent.com/assets/12813762/12816659/8c90e098-cb1b-11e5-8b33-0f7a6b77159a.png">
<img width="895" alt="screen shot 2016-02-04 at 8 43 54 am" src="https://cloud.githubusercontent.com/assets/12813762/12816658/8c8e9ea0-cb1b-11e5-8ba0-4ceef0964ca1.png">